arraycontext/context.py

```diff
@@ -299,9 +299,19 @@ def zeros(self,
         pass
 
     def empty_like(self, ary: Array) -> Array:
+        from warnings import warn
+        warn(f"{type(self).__name__}.empty_like is deprecated and will stop "
+            "working in 2023.",
```

Comment:
Should this recommend some replacement or something? Nothing came to mind..

Reply:
Why not `actx.np.empty_like`?

Reply:
Ah! `empty`! Sorry, lost the plot for a second. Maybe suggest `zeros_like`?

Reply:
Sure, went with that in https://github.com/inducer/arraycontext/compare/b2785c47586f3b0323a5ce27684a2708756cecd2..7283002dee5013b6f1e87c24c2e9335a6b54913c.